### DISCUSSION TITLE
Binding EnOcean PTM 215Z to Sonoff ZBMINI, how?

### DISCUSSION THREAD

**Discussion by m0wlheld** [Koenkk/zigbee2mqtt#10638] [Devices]
Trying to replace some in-wall Homematic switch/actors [HM-RC-2-PBU-FM](https://www.eq-3.com/products/homematic/detail/homematic-wireless-remote-control-2-channel-for-brand-switch-systems-flush-mount.html), as I never managed to get Homematic to work realiable. The benefit of those Homematic devices  is however, that they work even without a gateway / coordinator for manual use.

I though I could minic this behaviour by binding an EnOcean PTM 215Z (inside a "Scenic Friends of Hue" rocker) to a Sonoff ZBMINI. This way, it should work even if Z2M or associated Home Assistant fails.

However, when I select the EnOcean device and go to the "Binding" tab, there's not much to do. I can only select the source's endpoint, the target and the target's endpoint. No clusters are displayed and the "bind" action button ("Verbinden") is disabled. 

Am I doing wrong or have I selected inappropriate hardware?

![grafik](https://user-images.githubusercontent.com/23745709/148547595-fef3c622-b22c-4cbd-a1a3-9ebaba866e24.png)

**Comment by Koenkk** [comment]
Green power devices cannot be bound

**Reply by m0wlheld** [reply]
👍 for the quick reply, 👎 for the device. As I understand, it's only getting worse: Not only do Z2M and Home Assistant be running, but also a Hue bulb (I have a few) must be close as well. 3 points of failure that can turn my smart home into a dumb, dark cave ..

**Comment by Danit2** [comment]
Do you have read this:

https://www.zigbee2mqtt.io/devices/PTM_215Z.html#green-power

They can not be bind to other device. 
You must do it over HA or other. 
I do it over Node-Red.

**Reply by m0wlheld** [reply]
Obviously I haven't, otherwise I wouldn't have spent 69 EUR on a not-so-smart-rocker :(

**Reply by yvansandoz** [reply]
Any chance to see a sample of one of your node-red automation with PTM 215Z switches connected with zigbee2mqtt?

**Reply by Danit2** [reply]
![Screenshot 2022-02-12 194500](https://user-images.githubusercontent.com/71522810/153724204-fb99a138-bef8-4c61-8082-6839ad0d6342.png)
![Screenshot 2022-02-12 194538](https://user-images.githubusercontent.com/71522810/153724206-22264671-49c2-428f-ab72-878314949ebc.png)
![Screenshot 2022-02-12 194709](https://user-images.githubusercontent.com/71522810/153724207-656b0b56-c66b-43c5-80cc-6dd587d5f251.png)

I use the two upper button to switch on the light and the two under to switch off.
When you press both upper together the light will dim up and the other two toghether will dim down. 

Here you have the Flow:
[flows (1).zip](https://github.com/Koenkk/zigbee2mqtt/files/8054293/flows.1.zip)

**Reply by yvansandoz** [reply]
Thanks a lot. By the way, I found a nice add-on in node-red ( button-events) in order to manage long press, which does add another layer of options for your scheme.

**Comment by m0wlheld** [comment]
Are you aware on any device being similar to the Homematic actors above that are suitable for the German market? Most of the devices on the supported devices list are designed to be a single switch item on the wall, but in Germany we often use multiple switches / rockers inside a single frame, with ~55mm for the actual key and ~80mm for the frame (like [Merten M1](https://www.merten.de/produkte/schalterprogramme/system-m/1-m.html) or Gira or Busch Jaeger).

**Comment by Danit2** [comment]
I have the same problem. 
I will have  a zigbee network that work also when Z2M is down. 

I use the Ubisys devices. 
There are very cool. 
Like this:

https://www.zigbee2mqtt.io/devices/C4.html#ubisys-c4

But also the dimmer works great. 
I use my old pushbuttons so i do not have a problem with the design.

**Reply by sjorge** [reply]
+1 the Ubisys C4 is great and I also have a bunch of S1, S2, and J1’s.

Doing it all over again I’d just have used a S2 everywhere though, the switching is silent unlike the S1.

**Comment by m0wlheld** [comment]
Thanks, but "The ubisys C4 remote control unit seems to be primarily targeted to be directly bound to other ZigBee devices to control them." does not match my use case. Currently looking a the "Wiser" system of Schneider Electric / Merten, but their documentation is completely unclear about what is needed and what technology is used.

**Reply by Danit2** [reply]
What is your use case? 
You can use the ubisys also to control other things over MQTT and/or HA.

**Comment by m0wlheld** [comment]
I want to be able to switch the lights manually, even if the coordinator hardware or software is broken. 

You can connect a „classic“ momentary switch to the ZBMini, but unfortunally there‘s not enough room in the wallbox (Unterputzdose).

**Reply by Danit2** [reply]
https://www.zigbee2mqtt.io/devices/S2.html#ubisys-s2

This is exactly what ubisys do. 
Do you have see all the device from ubisys. 
There are devices with:
One switch and one input
One dimmer and two input
Four inputs
Two switch and two input
And more. 

When you bind the first input to the first output the the hole network can crash and you can control the light localy.

**Comment by m0wlheld** [comment]
I appreciate your feedback and Ubysis and so far I have nothing but positive on Ubysis  components. But:

- you get switch functionality for 1/4 of the price using a Sonoff ZBMINI which is even smaller than the S1/S2 devices
- still i don't have enough space in the flush mount box for a ZBMini / S1 *and* a standard rocker or switch

**Reply by Danit2** [reply]
> * you get switch functionality for 1/4 of the price using a Sonoff ZBMINI which is even smaller than the S1/S2 devices

But by ubisys you have quality. 

> * still i don't have enough space in the flush mount box for a ZBMini / S1 _and_ a standard rocker or switch

I don't know what kind of mount boxes you have, but in switzerland you can have a "smal combination" with a switch and a plug in one box. There you can have problem with the space. but i have also there a D1 behind in a normal 55mm box.
